app/assets/stylesheets/abstracts/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$blue: #0139A6;
+$white: #EBF5EE;
+$yellow: #F9C80E;
+$green: #43A047;
+
+// Colores para la tabla
+$color-fuente: #333;
+$color-fondo-cabecera: #6baed6;
+$color-fuente-cabecera: #fff;
+$color-borde: #ddd;
+$color-fila-alterna: #f2f2f2;
+$color-hover: #ddd;
+
+
+$breakpoints-sm: 799px;
+$breakpoints-md: 800px;
+$breakpoints-lg: 1000px;
+$breakpoints-xl: 1200px;
```

Comment:
set these variables in `_mixins.scss` file to manage in an easy way these breakpoints.

Reply:
done  :+1: